### DISCUSSION TITLE
IllegalStateException: Could not retrieve TraCI API version. 	-Root Cause: SocketException: Connection reset

### DISCUSSION THREAD

**Discussion by adrs221** [eclipse/mosaic#60] [Q&A] [answered]
I am running a sumo simulation using mosaic. When I specify in the .sumocfg file which output files I need, I get the following error. From what I have been able to find online, it is a server-side error. It seemed in another question on this discussion page that all I needed to do was specify my output files in the sumocfg file but is there something else I have to do? I've looked in the mosaic documentation but am still very stuck on how to proceed. Thank you.

>  Stopping simulation due to a critical error:
> 	- IllegalStateException: Could not retrieve TraCI API version.
> 	- Root Cause: SocketException: Connection reset
>  Please see the log files for details.
> 	- Log-Directory: logs/log-20210308-224552-test
> 
>  Eclipse MOSAIC will now shutdown.

**Comment by kschrab** [comment, marked as answer]
Have you looked for an error stated in the log-files? You may look into `Traffic.log`. I would guess that something with your `sumocfg` file is wrong.

**Reply by adrs221** [reply]
Got it! The path for where to put the output files was invalid. It's working fine now. Thank you!
